### PR TITLE
Improve code style in deploy_linux.sh

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Smaller additions/modifications to deploy_linux.sh (also based on spellcheck proposals). Now the script needs to be run from the repo-root (similar to some other scripts).

CHANGELOG: SKIP

**Context: Fixes an issue?**
No. Just a few less warnings if we include spell-check in our CI. 

**Does this change need documentation? What needs to be documented and how?**
No. I don't think it's even ChangeLog worthy since it could be added to the refactoring step

**Status of this Pull Request**
Working implementation

**What is missing until this pull request can be merged?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

File: linux/deploy_linux.sh [hunk 1]
```diff
@@ -1,11 +1,13 @@
-#!/bin/sh
```
Comment (ann0see) [inline linux/deploy_linux.sh:1]: I'm not even sure who uses this script.

Comment (pljones) [reply linux/deploy_linux.sh:1]: Probably the old Sourceforce distro.  Pre-`.deb`.  Basically it's the "flavourless" Linux option.

Comment (hoffie) [reply linux/deploy_linux.sh:1]: > I'm not even sure who uses this script.

I wondered about that as well. Personally, I'd say, delete it, but maybe others have more information about the history.

Comment (pljones) [reply linux/deploy_linux.sh:1]: Does anything else use the 
> `make dist`

artifacts?

Anyway, in a similar way to having a "Linux .deb" distribution, this provides a "Linux .tar.gz" distribution - but with no dependency information embeded (as it's not a .deb or .rpm, etc).

Comment (ann0see) [reply linux/deploy_linux.sh:1]: Just tried it - it seems to include the source - I don't think we're planning to release the tar anymore. Linux users will be able to just git clone the repo.

Comment (pljones) [reply linux/deploy_linux.sh:1]: Hm, I'd be inclined just to delete it with a note in the changelog to that effect, then.

Comment (ann0see) [reply linux/deploy_linux.sh:1]: Ok. Closing this issue then.

**Comment by ann0see** [comment]
Closed as it makes sense to delete this script (it's no longer in use)

**Pull request closed without merging**